SPECS/mstflint/mstflint.spec

```
@@ -15,6 +17,8 @@ BuildRequires:  automake
 BuildRequires:  boost-devel
 BuildRequires:  gcc
 BuildRequires:  gcc-c++
+BuildRequires:  kernel-devel
```

Let's also include the version constraints for `kernel-devel` for the sake of consistency, being explicit, and the fact, that the BRs are what actually dictates the run-time requirements due to how RPM figures out implicit run-time dependencies.
```suggestion
BuildRequires:  kernel-devel >= 5.15.0
BuildRequires:  kernel-devel < 5.16.0
```